testservices/novaservice/service_http_test.go

```diff
@@ -18,30 +18,39 @@ import (
 	"gopkg.in/goose.v1/testing/httpsuite"
 	"gopkg.in/goose.v1/testservices/hook"
 	"gopkg.in/goose.v1/testservices/identityservice"
+	"gopkg.in/goose.v1/testservices/neutronmodel"
 )
 
 type NovaHTTPSuite struct {
 	httpsuite.HTTPSuite
-	service *Nova
-	token   string
+	service              *Nova
+	token                string
+	useNeutronNetworking bool
 }
 
-var _ = gc.Suite(&NovaHTTPSuite{})
+var _ = gc.Suite(&NovaHTTPSuite{useNeutronNetworking: false})
 
 type NovaHTTPSSuite struct {
 	httpsuite.HTTPSuite
-	service *Nova
-	token   string
+	service              *Nova
+	token                string
+	useNeutronNetworking bool
 }
 
-var _ = gc.Suite(&NovaHTTPSSuite{HTTPSuite: httpsuite.HTTPSuite{UseTLS: true}})
+var _ = gc.Suite(&NovaHTTPSSuite{HTTPSuite: httpsuite.HTTPSuite{UseTLS: true}, useNeutronNetworking: true})
```

Comment:
Yes, it is an arbitrary pick, however I saw no reason to double the number of tests to complete the matrix.  The two code paths are unrelated.